### ISSUE TITLE
Some documentation notes

### ISSUE THREAD

**Issue by kuraga** [mindee/doctr#1502]
> It's also open see for guidiance: https://mindee.github.io/doctr/using_doctr/custom_models_training.html
> 
> Only the mindee internal data which was used to train the models can't be shared because they contain sensible data :)

@felixdittrich92 , just quick notes
(don't know whether they are relevant, feel free to close):

1. The link above doesn't contain [`classification`](https://github.com/mindee/doctr/tree/main/references/classification), [`obj_detection`](https://github.com/mindee/doctr/tree/main/references/obj_detection).
2. The [`classification`](https://github.com/mindee/doctr/tree/main/references/classification)'s [`Data format'](https://github.com/mindee/doctr/tree/main/references/classification#data-format) part is a little bit strange: there are no any labels data.
3. The [`obj_detection`](https://github.com/mindee/doctr/tree/main/references/obj_detection) doesn't have `README.md`.

Also,

4. https://mindee.github.io/doctr/modules/models.html contains `a Keras model` statements.
5. `Args` there - are crumpled.

Thanks!

**Comment by kuraga** [comment]
> yeah you are right 👍 We are always happy about any kind of PR (including documentation fixes 😃)

Assume it's about pp. 4., 5. :smile:

**Comment by felixdittrich92** [comment]
> > yeah you are right 👍 We are always happy about any kind of PR (including documentation fixes 😃)
> 
> Assume it's about pp. 4., 5. 😄

Yep guidelines can be found here: https://mindee.github.io/doctr/contributing/contributing.html  ^^
